### PR TITLE
Fix Bug 1491154, switch from Verdana to Arial

### PR DESCRIPTION
**Pull request by schalkneethling**
This switches to Arial from Verdana. As a result I had to also make some tweaks to the typography. It is not a 1:1 match, but then it is not intended to be. Let me know your thoughts. Thanks!

### CODE REVIEW

File: kuma/static/styles/components/home/home-masthead.scss [hunk 1]
```diff
@@ -20,7 +20,7 @@ homepage masthead (search, feature callouts)
     }
 
     h1 {
-        @include set-font-size(36px);
+        @include set-font-size(42px);
```
Comment (ExE-Boss) [inline kuma/static/styles/components/home/home-masthead.scss]: Shouldn’t this be using `$h2-font-size` or&nbsp;another such variable?

Comment (schalkneethling) [reply kuma/static/styles/components/home/home-masthead.scss]: Well, yes and no ;) I am working on the side on a more modular scale for our typography with some meaningful class/variable names. It just feels strange to me to set the `font-size` for an `h1` to `$h2-font-size`

**Comment by jwhitlock** [comment]
@schalkneethling can you rebase? PR #5031 was merged first.

**Comment by schalkneethling** [comment]
@jwhitlock rebased.

File: kuma/static/styles/components/wiki/quick-links.scss [hunk 2]
```diff
@@ -37,7 +37,7 @@
     }
 
     li li {
-        @include bidi(((padding-left, $grid-spacing, padding-right, 0),));
+        @include bidi(((padding-left, $grid-spacing, padding-right, 0)));
```
Comment (ExE-Boss) [inline kuma/static/styles/components/wiki/quick-links.scss]: Looks like there’s [an error in calling this](https://travis-ci.org/mozilla/kuma/jobs/442037276#L2606).

https://github.com/mozilla/kuma/blob/f088e94455be79acae6bb3a9e932c18e3338db4d/kuma/static/styles/includes/_mixins.scss#L378-L383

Based on the above, this needs the trailing comma which you removed 🤦‍🏻‍♂️

Comment (schalkneethling) [reply kuma/static/styles/components/wiki/quick-links.scss]: Ooooh, bad rebase. Will fixup.

Comment (schalkneethling) [reply kuma/static/styles/components/wiki/quick-links.scss]: Oh no, it was prettier that removed the `,`

**Pull request merged**